www/video/videotest.js

```diff
@@ -21,6 +23,16 @@ function AddUrl()
 
 function RemoveUrl(id) 
 {
+    urlCount -= 1;
     $(id).remove();
     return false;
 }
+function showRemoveLink(){
```

Comment:
Try to follow pattern of case for functions in the rest of the file: showRemoveLink should be ShowRemoveLink.

Also I wonder if a more appropriate name could be UpdateRemoveLink.